### DISCUSSION TITLE
w2form URL not send POST request

### DISCUSSION THREAD

**Discussion by quagliarellamichel** [vitmalina/w2ui#2251] [Q&A]
hi guys
my configuration is as follows

```
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
<script src="https://rawgit.com/vitmalina/w2ui/master/dist/w2ui.min.js" type="text/javascript"></script>
```

I am trying to make a request to the server to update the list of client names
but the call to the URL doesn't work
i debugged the server to see if it would at least make an attempt but nothing

the code is as follows

```
new w2field('list', {
    el: query('#list')[0],
    url: '@Url.Action("FindCustomer", "Api")',
})
```

if someone helps me I'll buy him a beer

greetings

**Comment by vitmalina** [comment]
I am afraid, but currently list field in master branch does not have functionality to pull data from remote server. You can either use v 1.5, or wait a few weeks until it is completed. I refactored w2fields a month ago, and this piece was not completed yet. Also, you no longer need jQuery in 2.0 :)
